server/src/main/scala/controllers/Application.scala

```diff
@@ -173,8 +173,8 @@ class Application @Inject()(implicit val config: Configuration,
 
   log.info(s"System.getProperty(user.timezone): $systemTimeZone")
   log.info(s"TimeZone.getDefault: $defaultTimeZone")
-  assert(systemTimeZone == "UTC")
-  assert(defaultTimeZone == "UTC")
+//  assert(systemTimeZone == "UTC", "System Timezone is not set to UTC")
```

Comment:
we prob want these uncommented right? i like the message addition though

Reply:
fixed.